cmd/nfd-topology-updater/main.go

```diff
@@ -109,8 +117,8 @@ func initFlags(flagset *flag.FlagSet) (*topology.Args, *resourcemonitor.Args) {
 		"Time to sleep between CR updates. Non-positive value implies no CR updatation (i.e. infinite sleep). [Default: 60s]")
 	flagset.StringVar(&resourcemonitorArgs.Namespace, "watch-namespace", "*",
 		"Namespace to watch pods (for testing/debugging purpose). Use * for all namespaces.")
-	flagset.StringVar(&resourcemonitorArgs.KubeletConfigFile, "kubelet-config-file", source.VarDir.Path("lib/kubelet/config.yaml"),
-		"Kubelet config file path.")
+	flagset.StringVar(&resourcemonitorArgs.TokenFile, "token-file", "",
```

Comment:
I would prefer to add this feature - not to replace the previous. If things turn well, we will deprecate `kubelet-config-file` over time.

Reply:
Good idea, I will try to use token first. If failed, we could fall back to use config file.